toolchain/sem_ir/ids.h

```diff
@@ -268,7 +278,7 @@ struct ImplId : public IdBase, public Printable<ImplId> {
 
 constexpr ImplId ImplId::Invalid = ImplId(InvalidIndex);
 
-// The ID of an imported IR.
+// The ID of an IR within the set of imported IRs, both direct and indirect.
```

Comment:
Is this comment right? Looking at `InitPackageScopeAndImports`, it looks to me like we only assign `ImportIRId`s to directly imported IRs.

Reply:
Oh, I see this changes in the next PR.